### DISCUSSION TITLE
specify which queues worker should listen to?

### DISCUSSION THREAD

**Discussion by aon108** [hibiken/asynq#361] [Q&A]
Could you please clarify if a worker can listen only to particular queues?
example: when client enqueues a taskA it specifies a queue name queA. 

- One worker then specifies that same queue name queA in the Config.
- Another worker (different instance) specifies queB in it's config.

cheers

**Comment by hibiken** [comment]
Queues are created dynamically in asynq so if you provide the `Queue(qname)` option for a task and if the queue with the given name doesn't exist yet, it will create that queue.

Also multiple worker instances can consume tasks from the same queue, a given task will delivered to only one instance at a time.
Let me know if this clarify things.
